src/js/utils/api.js

```diff
@@ -58,7 +58,8 @@ module.exports.call = function call(options) {
 
     var headers = {
         'Accept': 'application/json',
-        'Content-Type': 'application/json'
+        'Content-Type': 'application/json',
+        'x-smooch-sdk': 'web/' + Smooch.VERSION
```

Comment:
hmmm that makes sense... I'd say, set the version in endpoint and get it from there instead.